containers/codespaces-linux/.devcontainer/Dockerfile

```diff
@@ -85,39 +85,39 @@ RUN apt-get update -yq \
     # Install Helm
     && curl -s https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash - \
     #
-    # Setup vsonline user
+    # Setup codespace user
     && { echo && echo "PATH=${EXTRA_PATHS_OVERRIDES}:\$PATH:${USER_EXTRA_PATHS}" ; } | tee -a /etc/bash.bashrc >> /etc/skel/.bashrc \
     && { echo && echo $BASH_PROMPT ; } | tee -a /etc/bash.bashrc >> /etc/skel/.bashrc \
     && printf "$FISH_PROMPT" >> /etc/fish/conf.d/fish_prompt.fish \
     && { echo && echo $ZSH_PROMPT ; } | tee -a /etc/zsh/zshrc >> /etc/skel/.zshrc \
-    && useradd --create-home --shell /bin/bash vsonline \
+    && useradd --create-home --shell /bin/bash codespace \
     && mkdir -p /etc/sudoers.d \
-    && echo "vsonline ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/nopasswd \
+    && echo "codespace ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/nopasswd \
     && echo "Defaults secure_path=\"/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin:${EXTRA_PATHS}\"" >> /etc/sudoers.d/securepath \
-    && sudo -u vsonline mkdir /home/vsonline/.vsonline \
+    && sudo -u codespace mkdir /home/codespace/.vsonline \
     && groupadd -g 800 docker \
-    && usermod -a -G docker vsonline \
+    && usermod -a -G docker codespace \
     #
     # Setup .NET Core
     # Hack to get dotnet core sdks in the right place - Oryx images do not put dotnet on the path because it will break AppService.
-    # The following script will put the dotnet's at /home/vsonline/.dotnet folder where dotnet will look by default.
-    && mv /tmp/vsonline/symlinkDotNetCore.sh /home/vsonline/symlinkDotNetCore.sh \
-    && sudo -u vsonline /bin/bash /home/vsonline/symlinkDotNetCore.sh 2>&1 \
-    && rm /home/vsonline/symlinkDotNetCore.sh \
+    # The following script will put the dotnet's at /home/codespace/.dotnet folder where dotnet will look by default.
```

Comment:
Have we made corresponding changes in Cascade repo?

Reply:
Yep the first part is already merged in.